lib/generators/notify_user/install/templates/create_notify_user_notifications.rb

```diff
@@ -2,12 +2,11 @@ class CreateNotifyUserNotifications < ActiveRecord::Migration
   def change
     create_table :notify_user_notifications do |t|
       t.string :type
-      t.integer :target_id
-      t.string :target_type
       t.json :params
-      t.string :state
-      t.datetime :sent_time
+      t.datetime :read_at
 
+      t.integer :target_id
+      t.string :target_type
       t.integer :group_id
       t.integer :parent_id
```

Comment:
How do you go about moving from a previous version of NU to this one? Will we need a different migration?